package.json

```diff
@@ -33,10 +33,10 @@
     "postinstall": "patch-package"
   },
   "dependencies": {
-    "expo": "~36.0.0",
+    "expo": "~36.0.2",
     "react": "~16.9.0",
     "react-dom": "~16.9.0",
-    "react-native": "https://github.com/expo/react-native/archive/sdk-36.0.0.tar.gz",
+    "react-native": "https://github.com/expo/react-native/archive/sdk-36.0.1.tar.gz",
```

Comment:
I've had problems building the final IPA/APK in the past trying to update these 2 deps (`expo` and `react-native`). Something like the expo builder expected a different version. Did you find any documentation around this?

Reply:
Nevermind, all is good. The sdk version inside `app.json` is the finicky one